server/server.go

```diff
@@ -64,10 +67,12 @@ const (
 	serverMetricsInterval = time.Minute
 	leaderTickInterval    = 50 * time.Millisecond
 	// pdRootPath for all pd servers.
-	pdRootPath      = "/pd"
-	pdAPIPrefix     = "/pd/"
-	webPath         = "/web/"
-	pdClusterIDPath = "/pd/cluster_id"
+	pdRootPath       = "/pd"
+	pdAPIPrefix      = "/pd/"
+	webPath          = "/web/"
+	dashboardUIPath  = "/dashboard/"
+	dashboardAPIPath = "/dashboard/api/"
+	pdClusterIDPath  = "/pd/cluster_id"
```

Comment:
```suggestion
	pdClusterIDPath  = "/pd/cluster-id"
``` Look like more restful

Reply:
This is not a change introduced by this PR.